backend/fms_core/services/profile.py

```diff
@@ -0,0 +1,54 @@
+from collections import defaultdict
+from fms_core.models import Profile, FreezemanUser
+from fms_core.schema_validators import PREFERENCES_VALIDATOR
+
+def get_default_profile() -> Profile:
+    return Profile.objects.get(name="Default")
+
+def get_profile(user_id: int) -> Profile:
+    from fms_core.services.freezeman_user import get_or_create_freezeman_user
+    fm_user = get_or_create_freezeman_user(user_id)
```

Comment:
I am not sure we want this by freezeman_user_id ... The frontend should be unaware of that id so will not be providing it ... there is not much added value to use that id here over the django user id.